cmd/snap-bootstrap/partition/sfdisk.go

```diff
@@ -57,6 +68,7 @@ type sfdiskPartition struct {
 	Node  string `json:"node"`
 	Start uint64 `json:"start"`
 	Size  uint64 `json:"size"`
+	Attrs string `json:"attrs"`
```

Comment:
Attrs is actually quite opaque, can you add an example in the comments here, explaining the meaning the fields? I undernstand it's something like `[<someknownattr>,]GUID:<set-bit>[,<set-bit-n>]`

Reply:
Named attributes before the numeric GUID:x,y are not being correctly handled. Fixing it.

Reply:
Funny how manpage leaves out some details:
```
The attributes argument is a comma or space-delimited list of bits.  
The currently supported attribute bits are: RequiredPartition,
NoBlockIOProtocol, LegacyBIOSBootable  and  GUID-specific bits
in the range from 48 to 63. For example, the string
"RequiredPartition,50,51" sets three bits.
```

Then in the command line:
```
$ sfdisk --part-attrs disk 2 'RequiredPartition,NoBlockIOProtocol,58,59,60'
$ sfdisk --json disk | jq '.partitiontable.partitions[1].attrs'
"RequiredPartition NoBlockIOProtocol GUID:58,59,60"
```
Perhaps it'd be useful to leave a note that the even though the manpage says `--part-attrs` takes space or comma separated list, the `--json/--dump` uses `<attr> [<attr>]` format with `GUID:<bit>[,<bit>]`